spec/requests/api/members_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe "api/members" do
+
+  def json
+    JSON.parse(response.body)
+  end
+
+  describe 'POST api/members' do
+    let(:new_member) { build :member, email: "newbie@test.org" }
```

Comment:
You don't need `new_member` for this spec. You just want an email address, right?